ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -601,14 +607,56 @@ public List<String> availableImports() {
     public List<String> availableVariables() {
         // Available variables and values as string.
         List<String> varStrings = new ArrayList<>();
+        List<String> variablesDeclarations = new ArrayList<>();
+        List<String> finalVariablesDeclarations = new ArrayList<>();
+        List<String> constVariablesDeclarations = new ArrayList<>();
```

Comment:
```suggestion
        List<String> constDeclarations = new ArrayList<>();
```

Reply:
Fixed in cd59992ad5c45fb100076e2cd13154f643ae153b